cpp/tests/device_atomics/device_atomics_test.cu

```diff
@@ -92,26 +92,6 @@ void gpu_atomic_bitwiseOp_test(T *result, T *data, size_t size)
     }
 }
 
-// TODO: remove these explicit instantiation for kernels
-// At TYPED_TEST, the kernel for TypeParam of `wrapper` types won't be instantiated
-// because `TypeParam` is a private member of class ::testing::Test
-// then the kenrel call fails with `cudaErrorInvalidDeviceFunction`
-
-template  __global__ void gpu_atomic_test<cudf::date32>(cudf::date32 *result, cudf::date32 *data, size_t size);
```

Comment:
What made it possible to remove these now?

Reply:
Yes, actually we can remove these.
At first it was required because the kernel is called from derived class method of `AtomicsTest ` using private ParamType.
For now, the kernel call is moved to a method of `AtomicsTest ` using the public template parameter T.